tests/installation/installation_overview.pm

```diff
@@ -79,6 +79,16 @@ sub check_default_target {
     if (my $log_line = script_output("grep '$target_search' /var/log/YaST2/y2log | tail -1",
             proceed_on_failure => 1)) {
         $log_line =~ /$target_search: (?<current_target>.*)/;
+
+        # Default target is set to graphical if VNC is used during installation bsc#1167736
+        if (check_var('REMOTE_CONNECTION', 'vnc') && check_var('ARCH', 's390x') && check_var('DESKTOP', 'textmode')) {
```

Comment:
I would go for adding new variable referencing the bug and set it in these two scenarios, instead of 4 conditions.
Once it is merged, you could mentioned in the bug that you added the soft-failure so other team facing the same issue(I think it was QAM) can also add it.

Reply:
At least, we don't need special condition for s390x, as it's same on other remote backends.
We also can simplify it to just recording soft failure in case of mismatch and rework this check once outcome in the bug is clarified. E.g. we set expectation based on which installation is used.

Reply:
> I would go for adding new variable referencing the bug and set it in these two scenarios, instead of 4 conditions.
> Once it is merged, you could mentioned in the bug that you added the soft-failure so other team facing the same issue(I think it was QAM) can also add it.

I thought of it, but that approach also has the issue that we don't actually verify that the problem happens, this leads to have some workarounds still active long after the problem is solved, spamming bug reports for no reason. At least I'd keep that one `if ($+{current_target} eq "graphical")`.

Reply:
Also, that problem is very specific. It does not happen on other backends.

Reply:
though probably we could effectively remove the check for s390x. as far as I understood, it happens when we have VNC and that DESKTOP is not defined.

Reply:
It gets defined as per system role and for SLES 15 default is textmode (no desktop applications module). Happens in `main.pm` somewhere.